src/styles/responsive.js

```diff
@@ -1,30 +1,102 @@
-const COLUMN_WIDTH =    104               // 1/2 Item Card width
+const COLUMN_WIDTH = 104                  // 1/2 Item Card width
 
-const SCREEN_SM_MIN =   768               // 6 columns rounded
-const SCREEN_MD_MIN =   COLUMN_WIDTH * 8  // 832
+const SCREEN_SM_MIN = 768                 // 6 columns rounded
+const SCREEN_MD_MIN = COLUMN_WIDTH * 8    // 832
 const SCREEN_MDLG_MIN = COLUMN_WIDTH * 10 // 1040
-const SCREEN_LG_MIN =   COLUMN_WIDTH * 12 // 1248
-const SCREEN_XL_MIN =   COLUMN_WIDTH * 14 // 1456 (max width)
+const SCREEN_LG_MIN = COLUMN_WIDTH * 12   // 1248
+const SCREEN_XL_MIN = COLUMN_WIDTH * 14   // 1456 (max width)
 
 const SCREEN_XS_MAX = SCREEN_SM_MIN - 1   // 767
 const SCREEN_SM_MAX = SCREEN_MD_MIN - 1   // 831
 const SCREEN_MD_MAX = SCREEN_MDLG_MIN - 1 // 1039
 const SCREEN_MDLG_MAX = SCREEN_LG_MIN - 1 // 1247
 const SCREEN_LG_MAX = SCREEN_XL_MIN - 1   // 1455
 
+// TODO: exporting just for test (so far at least), smell?
+export const sizes = {
+  xs: {
+    // TODO: no min, hmmm
```

Comment:
maybe 0? Does having a `min` help?

Reply:
👍 thank, I went with this. Having a `min` helps avoid having to add a few special cases.